NEWS.md

```diff
@@ -1,4 +1,17 @@
-## 1.8.0 Unreleased
+## 1.9.0 Unreleased
```

Comment:
Next `Unreleased` should be `1.8.1` ?

Reply:
We usually use the following guide https://semver.org. X.Y.Z - Z means bug fix version.